assets/scripts/admin/labels.js

```diff
@@ -110,6 +111,9 @@ function quickaddfunction() {
     }
 
     console.log('lsrows', lsrows);
+
+    // parsedRows is an array of objects with the parsed row data (ie: [{code:"code", titles:{"en":"Title"}}])
+    let parsedRows = [];
```

Comment:
Nice with example data in comment. Maybe we can even create types here one day (TypeScript or Flow, for example).